src/pages/[slug].tsx

```diff
@@ -40,13 +40,13 @@ export const getServerSideProps = async ({ locale, params, query }: CustomNextPa
     const pageData = await useCtfPageQuery.fetcher({ slug, locale, preview })();
     const page = pageData.pageCollection?.items[0];
 
-    const topSection = page?.topSectionCollection?.items;
-    const extraSection = page?.extraSectionCollection?.items;
+    // const topSection = page?.topSectionCollection?.items;
```

Comment:
Mogen de comments weg, of heb je ze nog nodig?

Reply:
Ik dacht dat we ze later wel nodig konden hebben, hiermee haal je de data op van de top en extra section van de landing page